Readme.md

```diff
@@ -237,6 +237,85 @@ Your attributes will be automatically saved when you end the session, but if the
 
 We have wrapped up the above example into a high/low number guessing game skill you can [download here](https://github.com/alexa/skill-sample-nodejs-highlowgame).
 
+### Additional Event Processing
+
+The `OnRequest` event is similar to the `NewSession` event but is called regardless of the value of `event.session.new`. If you define `OnRequest` you cannot define `NewSession`. In fact all events will go to `OnRequest` and you will need to call another emit to have processing continue:
+
+```javascript
+var startGameHandlers = Alexa.CreateStateHandler(states.STARTMODE, {
+
+    'OnRequest': function () {
+        if (this.event.session.new) {
+            this.emit('NewSession'); // Uses the handler in newSessionHandlers
+        } else {
+            // Process request when session is not new
+        }
+        
+    },
+
+    // The OnRequest code above is equivalent to following event using NewSession:
+
+    //'NewSession': function () {
+    //    this.emit('NewSession'); // Uses the handler in newSessionHandlers
+    //},
+```
+
+Another important pattern with `OnRequest` is calling `this.emit(':forward')`:
```

Comment:
Could you give me a use case where you'd perfer to use :forward over emitWithState ?